infos/anfi-faq.md

```diff
@@ -85,7 +85,7 @@ Er ist nicht zwingend erforderlich, aber sehr nützlich, wenn man sich
 mit Mathe sowieso etwas schwer tut. Er fängt 2,5 Wochen vor dem Semester
 an und dauert zwei Wochen. Eine Anmeldung ist hierfür nicht
 erforderlich, nähere Informationen entnehmt ihr am besten dem
-[Anfängerbrief]().  
+[Anfängerbrief](https://github.com/fsi-tue/anfibrief/tree/master/pdf) oder dem Anfängerheft für [*Informatiker](https://github.com/fsi-tue/anfiheft/blob/master/anfiheft-info.pdf) oder [Kognis](https://github.com/fsi-tue/anfiheft/blob/master/anfiheft-kogni.pdf).  
```

Comment:
You should replace the links for Anfibrief pointing to the PDFs with the ones hosted on our infrastructure. See https://teri.fsi.uni-tuebingen.de/anfibrief/ . 
For now we can use the link https://www.fsi.uni-tuebingen.de/erstsemester/material/start for Anfiheft, we change the link in the future when we have CI/CD for Anfiheft so it is also hosted on https://teri.fsi.uni-tuebingen.de